src/gg.regression.unity.bots/Runtime/Scripts/StateRecorder/Models/RecordedGameObjectState.cs

```diff
@@ -76,8 +76,6 @@ public void WriteToStringBuilder(StringBuilder stringBuilder)
             Vector3JsonConverter.WriteToStringBuilderNullable(stringBuilder, position);
             stringBuilder.Append(",\n\"rotation\":");
             QuaternionJsonConverter.WriteToStringBuilderNullable(stringBuilder, rotation);
-            // TODO: Someday remove these no longer used fields
-            stringBuilder.Append(",\n\"rigidbodies\":[],\n\"colliders\":[],\n\"behaviours\":[]");
```

Comment:
I'll make sure to mention these in the changelog

Reply:
This information is still contained in the `components`, is this right?

Reply:
Correct, components is essentially all of these concepts + ECS data collapsed into one list

Reply:
@RG-nAmKcAz understood, I won't mention it, I just want us to get in the habit of making sure to make deprecation notes. Since this one likely didn't affect anyone anyways, I'll leave it out.